Formula/metricbeat.rb

```diff
@@ -0,0 +1,81 @@
+class Metricbeat < Formula
+  desc "Collect metrics from your systems and services."
+  homepage "https://www.elastic.co/products/beats/metricbeat"
+  url "https://github.com/elastic/beats/archive/v5.0.0.tar.gz"
+  sha256 "3e6b7cf2ee5f52e78ae87ef04ab9dd49977c89f86a09416586896aeaea844e34"
+
+  head "https://github.com/elastic/beats.git"
+
+  bottle do
+    cellar :any_skip_relocation
+  end
+
+  depends_on "go" => :build
+
+  def install
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/elastic/beats").install Dir["{*,.git,.gitignore}"]
+
+    ENV["GOPATH"] = gopath
+
+    cd gopath/"src/github.com/elastic/beats/metricbeat" do
+      system "make"
+      libexec.install "metricbeat"
+
+      (etc/"metricbeat").install "metricbeat.yml"
+      (etc/"metricbeat").install "metricbeat.template.json"
+      (etc/"metricbeat").install "metricbeat.template-es2x.json"
+    end
+
+    (bin/"metricbeat").write <<-EOS.undent
+      #!/bin/sh
+      exec "#{libexec}/metricbeat" --path.config "#{etc}/metricbeat" --path.home="#{prefix}" --path.logs="#{var}/log/metricbeat" --path.data="#{opt_prefix}" "$@"
+    EOS
+  end
+
+  plist_options :manual => "metricbeat"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN"
+    "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>Program</key>
+        <string>#{opt_bin}/metricbeat</string>
+        <key>RunAtLoad</key>
+        <true/>
+      </dict>
+    </plist>
+  EOS
+  end
+
+  test do
+    log_file = testpath/"log"
+    touch log_file
+
+    (testpath/"metricbeat.yml").write <<-EOS.undent
+      metricbeat.modules:
+      - module: system
+        metricsets: ["load"]
+        period: 1s
+      output.file:
+        enabled: true
+        path: #{testpath}/data
+        filename: metricbeat
+    EOS
+
+    mkdir testpath/"data"
+    mkdir testpath/"logs"
```

Comment:
Implementation changed to `mkpath`

I removed the log file as it is not needed. I forgot to remove it as I copied the initial code from the filebeat Formula.